definitions/v7/cathoderaytube.yml

```
@@ -76,7 +76,7 @@ search:
     - path: torrents.php
   inputs:
     $raw: "{{ range .Categories }}filter_cat[{{.}}]=1&{{end}}"
-    searchtext: "{{ .Keywords }}"
+    searchtext: "{{ if .Query.IMDBID }}{{ .Query.IMDBID }}{{ else }}{{ .Keywords }}{{ end }}"
```

```suggestion
    searchtext: "{{ .Query.IMDBID }}"
    searchtext: "{{ .Keywords }}"
```

`andmatch` rows filter can then also be removed.

This means that `Film Title` can be searched for, but `Film Title 1080p` will return no results. However, currently with the `andmatch` filter, the latter returns no results in Jackett anyway (as 1080p isn't in the title). Up for debate which is preferable.

---

It seems `andmatch` looks at `table#torrent_table > tbody > tr[class^="torrent row"]:has(a[href*="action=download"])` which includes the tags for CRT. So, `Film Title 1080p` does currently work.

We could also do `searchtext: "{{ if .Query.IMDBID }}{{ .Query.IMDBID }} {{ .Keywords }}{{ else }}{{ .Keywords }}{{ end }}"`

In that case, a search with t=movie&imdbid=tt_xxxx returns all torrents for the movie tt_xxxx
A search with  t=movie&imdbid=tt_xxxx&q=1080p returns all 1080p torrents for the movie t_xxxx
and t=movie&q=Film+Title+1080p returns all torrents matching film title that are 1080p

---

Prowlarr and Jackett treat `andmatch` differently, part of which is that Prowlarr not only matches the title but also the description (but not the whole row) - https://github.com/Prowlarr/Prowlarr/blob/db9f06156425782c79a2cfce8d944d867a4be929/src/NzbDrone.Core/Indexers/IndexerBase.cs#L152-L165

So while it works for Prowlarr, it doesn't for Jackett.

Ideally, if a tracker provides a 'title only' search function, we'd use that, however CRT's titles aren't really all that helpful. I'm kinda torn, but I think your suggested alternative is the best bet.